fhir-persistence-jdbc/pom.xml

```diff
@@ -30,8 +30,6 @@
         <dependency>
             <groupId>${project.groupId}</groupId>
             <artifactId>fhir-examples</artifactId>
-            <version>${project.version}</version>
-            <scope>test</scope>
```

Comment:
why is this changing the scope?

Reply:
its not.  we're just inhereting the scope from the parent now.  i think.
but i'll put the `scope` element back in to make sure (its more clear this way anyway).

Reply:
added scope=test back